geniza/footnotes/models.py

```diff
@@ -100,6 +101,10 @@ class Source(models.Model):
     page_range = models.CharField(
         max_length=255, blank=True, help_text="Page range for article or book section."
     )
+    publisher = models.CharField(max_length=255, blank=True, help_text="Publisher name")
+    place_published = models.CharField(
```

Comment:
please add the new fields to:
- list of source fields that are editable in the admin site (and would you add source url to the list also?) — see https://github.com/Princeton-CDH/geniza/blob/main/geniza/footnotes/admin.py#L115-L124 
- list of csv fields for admin export (see https://github.com/Princeton-CDH/geniza/blob/main/geniza/footnotes/admin.py#L168-L183 )

Reply:
Forgot about this! 😅 Thanks.